README.md

```diff
@@ -107,3 +107,10 @@ Build the project:
 (cd build && ninja && ninja test)
 ```
 
+## Quirks
+
+- When building with ninja, the Go wrapper glue file go/goohrli/goohrli.a is built.
+  Currently there's a known bug: ninja sometimes doesn't detect that this file needs to be rebuilt when the C++ files it depends on are changed.
+  Therefore, sometimes `rm go/goohrli/goorhli.a` before running ninja is needed.
+- When running long-running `-calculate` runs with `go/bin/score/score.go` they might pause if the machine is running in power save mode and decides to start sleeping.
```

Comment:
I saw this happening with performance mode, too -- I'm trying to disable the progress indications in go to see if it is filling some buffers in the terminals